src/autopredictor/select_model.py

```diff
@@ -0,0 +1,21 @@
+def select_model(df_output, model_name):
+    """
+    Selects and returns the scores of a specified model from a DataFrame of model scores.
+
+    This function searches a DataFrame, typically generated by the show_all() function,
+    for a model matching the name being provided. If found, it returns the corresponding row 
+    of scores. Otherwise, it issues a "Model Not Found""Model Not Found" message.
```

Comment:
"Model Not Found" is written twice.
```suggestion
    of scores. Otherwise, it issues a "Model Not Found" message.
```